integrationExamples/gpt/pbjs_partial_refresh_gpt.html

```diff
@@ -82,6 +82,11 @@
                             publisherId: 'TO ADD',
                             adSlot: 'TO ADD@300x600'
                         }
+                    }, {
+                        bidder: 'fan',
+                        params: {
+                            placementId: 'TO ADD'
```

Comment:
That would mean that Facebook would always return with an ad at a very high CPM... so yes, if that would help with the demo.